14-microbiome-diversity.Rmd

```diff
@@ -68,36 +100,44 @@ ggpubr::ggarrange(plotlist = plots, nrow = 2, ncol = 3, common.legend = TRUE, le
 
 ## Beta diversity
 
-In comparison to alpha diversities, beta diversities describe inter-microbiome
-(dis-)similarites. Technically speaking, beta diversities are usually
-represented as `dist` objects, which contain triangular data describing the
-distance between each pair of samples. These distances can be subjected to
-multi-dimensional scaling, also known as PCoA, to retrieve reduced dimensions
-for further evaluation or visualization.
+Where alpha diversity focuses on community variation within a
+community (sample), beta diversity quantifies (dis-)similarites
+between communities (samples).
+
+Technically, beta diversities are usually represented as `dist`
+objects, which contain triangular data describing the distance between
+each pair of samples. These distances can be further subjected to
+ordination with methods such as multi-dimensional scaling (also known
+as PCoA) to retrieve reduced dimensions for further evaluation or
+visualization. [TODO we could here instead link to ordination chapter?]
```

Comment:
I think a link to a chapter in the OSCA might work best for now.

Reply:
ok